openslides/urls.py

```diff
@@ -1,52 +1,17 @@
 from django.conf.urls import include, patterns, url
+from django.views.generic import RedirectView
 
-from openslides.core.views import IndexView, ErrorView
-from openslides.users.views import UserSettingsView, UserPasswordSettingsView
+from openslides.core.views import IndexView
 from openslides.utils.rest_api import router
 
-handler403 = ErrorView.as_view(status_code=403)
```

Comment:
I think we have to keep the error views for the moment because we still have Django views which still can produce errors like 500 oder 403.